DSA_Session5/src/employee/EmployeeTest.java

```diff
@@ -0,0 +1,55 @@
+package employee;
+
+import static org.junit.Assert.*;
+
+import java.util.List;
+
+import org.junit.Test;
+
+/**
+ * Test class for DictionaryImp Class methods
+ * 
+ * @author Aman Gautam
+ * 
+ * Dated - 06/08/2019
+ */
+public class EmployeeTest {
+
+	@Test
+	public void testSortBySalary() {
+		LinkedList employeeList = new LinkedList();
+		Employee employeeObject = new Employee("A", 12000, 20);
+		employeeList.insert(employeeList, employeeObject);
+		Employee employeeObject1 = new Employee("B", 30000, 23);
+		employeeList.insert(employeeList, employeeObject1);
+		Employee employeeObject2 = new Employee("C", 6000, 26);
+		employeeList.insert(employeeList, employeeObject2);
```

Comment:
You can not pass the calling object in the method insert().

Reply:
calling object removed